src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -210,6 +210,9 @@ static gyroSpiDetectFn_t gyroSpiDetectFnTable[] = {
 #ifdef USE_GYRO_SPI_ICM20689
     icm20689SpiDetect,  // icm20689SpiDetect detects ICM20602 and ICM20689
 #endif
+#ifdef USE_ACCGYRO_LSM6DSO
```

Comment:
What is the reason for moving this around?

Reply:
Just prioritizing the sensors that are actually used in supported boards to reduce startup time. There are a number of boards coming out with LSM6DSO soon. The ones that follow are not shipping in any supported boards at this point so it doesn't make sense to have them before ones that will be.

Reply:
Makes sense.